<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMEManagerWidget</class>
 <widget class="QWidget" name="FMEManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Hierarchial Methods</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_1">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>ward</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>average</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>complete</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>single</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Metric</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string>euclidean</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButton_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>manhattan</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>   &gt;  </string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="toolButton_3">
     <property name="text">
      <string>   &lt;  </string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QToolButton" name="toolButton_4">
     <property name="text">
      <string>&lt;&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="4">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="toolButton_1">
     <property name="text">
      <string>&gt;&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Input Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Attributes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="4">
    <widget class="QListWidget" name="listWidget_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
